src/routers/handlers/payment_callback/payment.callback.ts

```diff
@@ -7,6 +7,7 @@ export class PaymentCallbackHandler extends GenericHandler {
     constructor() {
         super({
             title: "Payment Callback – GOV.UK",
+            viewName: "payment callback"
```

Comment:
This line needs a comma at the end.
```suggestion
            viewName: "payment callback",
```